.gitignore

```diff
@@ -150,4 +150,5 @@ cython_debug/
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 .idea/
-app/.env.db
+
+/app/.env.db
```

Comment:
hmm, slash at the beginning means the file is relative to the system root directory. here you should specify things relative to your project root

Reply:
done